src/shared/contractOptionsTitle.tsx

```diff
@@ -0,0 +1,5 @@
+export enum ContractOptionsTitle {
```

Comment:
are we not using this `src/enum/contractOptionsTitle.ts`?
Also let's rename to `contractTabsTitle.ts`

Reply:
removed `src/shared/contractOptionsTitle.tsx` duplicated due to pulling from old branch
renamed to `contractTabsTitle`